spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/core/GcpContextAutoConfiguration.java

```diff
@@ -28,10 +28,10 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-
 /**
- * Base starter for Google Cloud Projects. Provides defaults for {@link com.google.auth.oauth2.GoogleCredentials}.
- * Binds properties from {@link GcpProperties}.
+ * Base starter for Google Cloud Projects. Provides defaults for
```

Comment:
What drives you to make changes in this class?

Looks like just code style polishing. And I even may believe that it is automatic by the IDE.

Can we avoid somehow to do this for nothing? 

Thanks